_pages/update-the-config-file/understanding-baseurl.md

```diff
@@ -21,6 +21,6 @@ production it's `https://pages.18f.gov/guides-template{{page.url}}`.
 
 ### Change the `baseurl:` when serving locally
 
-If you you do change the `baseurl:` property in the `_config.yml` file,
+If you do change the `baseurl:` property in the `_config.yml` file,
```

Comment:
"If you do you, change the `baseurl:` property in the `_config.yml` file."

![you-do-you-haters-gonna-hate](https://cloud.githubusercontent.com/assets/1244599/13687613/254561da-e6d1-11e5-8216-a902a73b3f3b.jpg)